modules/features/manatime_feature_config/manatime_feature_config.module

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @file
+ * Drupal needs this blank file.
+ */
+
+ /**
+  * Function to add aliases path.
+  */
+function add_path_alias($source, $alias, $language) {
+  $path = array(
+    'source' => $source,
+    'alias' => $alias,
+    'language' => $language,
```

Comment:
you could omit this. We should omit it because we don't want to fix stuff to a single language.